packages/rum-core/src/common/constants.js

```diff
@@ -175,6 +180,7 @@ export {
   BROWSER_RESPONSIVENESS_INTERVAL,
   BROWSER_RESPONSIVENESS_BUFFER,
   SIMILAR_SPAN_TO_TRANSACTION_RATIO,
+  TRANSACTION_DURATION_THRESHOLD,
```

Comment:
We can move this to performance monitoring file since it's only use there AFAICS.

Reply:
If we do that, we can also move all of the 4 there. I am okay with both.